### ISSUE TITLE
Document more detail about Fleet

### ISSUE THREAD

**Issue by catherineluse** [rancher/docs#3057]
Updated:
Request: Provide more detailed guidance on how to use the 'Continous Delivery' function for Fleet in Rancher. Explain how CD helps user queries and explain what "Force Updating" a Fleet cluster does?

Reference: https://rancher.com/docs/rancher/v2.6/en/deploy-across-clusters/fleet/#accessing-fleet-in-the-rancher-ui

Original issue:
The problem here is that it's an odd user experience in our documentation when people see that we link to Fleet docs, but then the docs on the Fleet website don't entirely apply to Rancher CD, and they ignore the Rancher UI. https://rancher.slack.com/archives/C6U4L2DS9/p1610632072001500

Denise said we should have descriptions of how to use Fleet in Rancher, but the examples and use cases should defer to Fleet.

**Comment by sirredbeard** [comment]
JIRA issue https://jira.suse.com/browse/SURE-3124?page=com.atlassian.jira.plugin.system.issuetabpanels%3Aall-tabpanel

**Comment by sirredbeard** [comment]
Thoughts on this, @jtravee?

**Comment by jtravee** [comment]
We are in the process of documenting more detailed examples/ use cases in Fleet as we speak. If you refer to the Fleet repo, there are many in the works now. I say if there are specific asks of how to use Fleet in Rancher, we will update the Rancher docs case-by-case. 

I would reword this particular docs issue to request specific additions to the Fleet docs in terms of CD and the questions users are asking (as is shown in the JIRA issue), e.g. Lack of clear guidance on how to use the 'Continous Delivery' views leads to support queries from users, e.g. 'What does "Force Update" a Fleet cluster do'?

I can then refer this to the devs to begin working on it.

**Comment by jtravee** [comment]
@StrongMonkey would you review and provide feedback?

**Comment by StrongMonkey** [comment]
@jtravee I am thinking that we can add a section in rancher fleet docs that teach user how to use fleet under rancher UI, as this is the missing part from fleet docs itself. The following can exist under `Accessing Fleet in the Rancher UI`.

```
1. Click ☰ > Continuous Delivery.
2. Select your workspace on top of the menu. By default it will be in `fleet-default`, which includes all downstream clusters that are registered through rancher. You can switch to `fleet-local` which only contains `local` cluster, or create your own workspace and assign/move clusters into it. You can manage clusters by clicking on `cluster` on the left colunm.
3. Click on Gitrepo on the left column to deploy gitrepo into your clusters in current workspace. Select your git repository and target clusters/cluster group. You can also create cluster group in UI. For more details, check https://fleet.rancher.io/gitrepo-add/ and https://fleet.rancher.io/gitrepo-structure/.
4. Once gitrepo is deployed you can monitor the application through rancher UI.
```
@jtravee Feel free to add some simple screenshot on each step to make it clear.
